tests/integration/tests.mk

```diff
@@ -41,41 +41,61 @@ else
 	_INTEGRATION_TEST_FLAGS += --istio.test.kube.config=~/.kube/config
 endif
 
-# Generate integration test targets for kubernetes environment.
+# Generate integration test targets for single-cluster kubernetes environment.
 test.integration.%.kube: | $(JUNIT_REPORT)
 	$(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
+	--istio.test.select -multicluster \
+	--istio.test.env kube \
+	${_INTEGRATION_TEST_FLAGS} \
+	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
+
+# Generate integration test targets for multicluster kubernetes environment.
+test.integration.%.kube: | $(JUNIT_REPORT)
+	$(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
+	--istio.test.select +multicluster \
 	--istio.test.env kube \
 	${_INTEGRATION_TEST_FLAGS} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
 # filter out non-standard test directories
 TEST_PACKAGES = $(shell go list ./tests/integration/... | grep -v /qualification | grep -v /examples)
 
-# Generate integration test targets for local environment.
+# Generate integration test targets for local environment. Only single-cluster tests are supported for local env.
 test.integration.%.local: | $(JUNIT_REPORT)
 	$(GO) test -p 1 ${T} -race ./tests/integration/$(subst .,/,$*)/... \
+	--istio.test.select -multicluster \
 	--istio.test.env native \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
-# Generate presubmit integration test targets for each component in kubernetes environment
+# Generate presubmit integration test targets for each component in single-cluster kubernetes environment
 test.integration.%.kube.presubmit: istioctl | $(JUNIT_REPORT)
 	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/integration/$(subst .,/,$*)/... -timeout 30m \
-	--istio.test.select -postsubmit,-flaky \
+	--istio.test.select -postsubmit,-flaky,-multicluster \
+	--istio.test.env kube \
+	${_INTEGRATION_TEST_FLAGS} \
+	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
+
+# Generate presubmit integration test targets for each component in multi-cluster kubernetes environment
+test.integration.%.kube.multicluster.presubmit: istioctl | $(JUNIT_REPORT)
```

Comment:
nit: should we just have this configured `INTEGRATION_TEST_FLAGS` (already defined) so we don't need more special targets?

Reply:
good suggestion. I've removed the extra targets and moved additional flags to `INTEGRATION_TEST_FLAGS`. PTAL